src/Operator/Constructors.jl

```diff
@@ -763,9 +906,19 @@ operator field:
     gradient
 ```
 """
-function advectionoperator(basis::AbstractBasis, mesh::Mesh, wind = [1, 1])
+function advectionoperator(
+    basis::AbstractBasis,
+    mesh::Mesh;
+    parameters::NamedTuple = (wind = [1.0, 1.0],),
+)
+    # check parameters
+    if !haskey(parameters, :wind)
+        parameters.wind = [1.0, 1.0] # COV_EXCL_LINE
+    end
```

Comment:
I was thinking of a check like this:

```suggestion
    end
    if basis.numbercomponents != 1
        throw(DomainError(basis.numbercomponents, "Advection gallery operator only accepts single component bases")) # COV_EXCL_LINE
    end
```

Reply:
Note that the SUPG constructors would also need this check

Reply:
yes, I am fixing that also in the SUPG constructors